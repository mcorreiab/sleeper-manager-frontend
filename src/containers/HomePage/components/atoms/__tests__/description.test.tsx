import renderer from "react-test-renderer";
import Description from "../description";

it("should render the description with text", () => {
  const tree = renderer.create(<Description />).toJSON();
  expect(tree).toMatchSnapshot();
});
