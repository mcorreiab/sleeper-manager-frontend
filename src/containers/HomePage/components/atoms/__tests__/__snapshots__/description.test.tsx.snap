// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the description with text 1`] = `
<p
  className="text-xl md:text-[1.4rem] font-['Open_Sans,sans-serif']"
>
  Insert your username to check all players with non ok injury status
</p>
`;
